### PR TITLE
Test and warning fixes

### PR DESCRIPTION
**Pull request by L-e-x-o-n**
Part 1 of fixes to different tests and warnings

### CODE REVIEW

File: lib/teiserver/telemetry/schemas/simple_server_event_type.ex [hunk 1]
```diff
@@ -11,6 +11,8 @@ defmodule Teiserver.Telemetry.SimpleServerEventType do
   """
   @spec changeset(Map.t(), Map.t()) :: Ecto.Changeset.t()
   def changeset(struct, params \\ %{}) do
+    IO.inspect(struct, label: "Struct:")
+    IO.inspect(params, label: "Params:")
```
Comment (geekingfrog) [inline lib/teiserver/telemetry/schemas/simple_server_event_type.ex:15]: I just noticed that, you probably don't want to commit these debug lines.

Comment (L-e-x-o-n) [reply lib/teiserver/telemetry/schemas/simple_server_event_type.ex:15]: I saw them yesterday as well and removed them locally already

**Comment by StanczakDominik** [comment]
So just to to clarify, was this fully obsoleted by #243?

**Comment by geekingfrog** [comment]
Doesn't look like it, not sure why this one was closed.

**Comment by L-e-x-o-n** [comment]
This has some different fixes and improvements but turned into a mess, I didnt have a spearate branch for it. Closed so I can clean it up and move to a new branch

**Pull request closed without merging**